galeracluster/source/documentation/mysql-wsrep-options.rst

```diff
@@ -182,6 +182,40 @@ You can execute the following ``SHOW VARIABLES`` statement to see how this varia
     | innodb-wsrep-applier-lock-wait-timeout | 10    |
     +----------------------------------------+-------+
 
+.. _`wsrep_applier_FK_failure_retries`:
+.. rst-class:: section-heading
+.. rubric:: ``wsrep_applier_FK_failure_retries``
+
+.. index::
+   pair: Parameters; wsrep_applier_FK_failure_retries
+
+Occasionally, foreign key constrains may fail even though the constraints themselves are not violated (for example, if the same transaction inserts in the parent table, and the next insert into the child table fails in FK checks). With this foreign key constraint check retrying implementation, you can control the number of retries. If the constraint check fails despite retires, the final retry prints out a warning with an error code and InnoDB system monitor output for further troubleshooting.
+
+.. csv-table::
+   :class: doc-options
+
+   "Command-line Format", "``--wsrep_applier_FK_failure_retries``"
+   "System Variable", "``wsrep_applier_FK_failure_retries``"
+   "Variable Scope", "Global"
+   "Dynamic Variable", "Yes"
+   "Permitted Values", "Integer"
+   "Default Value", "``1`` "
+   "Initial Version", "MySQL-wsrep 5.7.44"
```

Comment:
It only appeared in 8.0.35 not 5.7.44 please fix

Reply:
Fixed